File: GestionAssuranceFx/src/gestionassurancefx/Entities/OffreBonConducteur.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package gestionassurancefx.Entities;

import java.sql.Date;

/**
 *
 * @author ADMIN
 */
public class OffreBonConducteur extends Offre {
    private int numReglement;
    private int nbrAnnee;

    public OffreBonConducteur() {
        super(null, null, null, 0, null);
    }
    
    public OffreBonConducteur(int numReglement, int nbrAnnee,String libOffre, Date dateDebutOffre, Date dateFinOffre, int pourcentageReduction, String descripOffre) {
        super(libOffre, dateDebutOffre, dateFinOffre, pourcentageReduction, descripOffre);
        this.numReglement = numReglement;
        this.nbrAnnee = nbrAnnee;
    }
    public OffreBonConducteur(int idOffre,int numReglement, int nbrAnnee,String libOffre, Date dateDebutOffre, Date dateFinOffre, int pourcentageReduction, String descripOffre) {
        super(idOffre,libOffre, dateDebutOffre, dateFinOffre, pourcentageReduction, descripOffre);
        this.numReglement = numReglement;
        this.nbrAnnee = nbrAnnee;
    }

    

    public int getNumReglement() {
        return numReglement;
    }

    public int getNbrAnnee() {
        return nbrAnnee;
    }

    public void setNbrAnnee(int nbrAnnee) {
        this.nbrAnnee = nbrAnnee;
    }

    public void setNumReglement(int numReglement) {
        this.numReglement = numReglement;
    }

    
}

File: Assurance/src/AppBundle/Entity/Offrebonconducteur.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Offrebonconducteur
 *
 * @ORM\Table(name="offrebonconducteur")
 * @ORM\Entity
 */
class Offrebonconducteur
{
    /**
     * @var integer
     *
     * @ORM\Column(name="idOffre", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idoffre;

    /**
     * @var string
     *
     * @ORM\Column(name="libOffre", type="string", length=300, nullable=false)
     */
    private $liboffre;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateDebutOffre", type="date", nullable=false)
     */
    private $datedebutoffre;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateFinOffre", type="date", nullable=false)
     */
    private $datefinoffre;

    /**
     * @var integer
     *
     * @ORM\Column(name="pourcentageReduction", type="integer", nullable=false)
     */
    private $pourcentagereduction;

    /**
     * @var integer
     *
     * @ORM\Column(name="numReglement", type="integer", nullable=false)
     */
    private $numreglement;

    /**
     * @var integer
     *
     * @ORM\Column(name="nbrAnnee", type="integer", nullable=false)
     */
    private $nbrannee;

    /**
     * @var string
     *
     * @ORM\Column(name="descripOffre", type="string", length=100, nullable=false)
     */
    private $descripoffre;


}

